ConcernsCaseWork/ConcernsCaseWork.Data/Migrations/ConcernsDbContextModelSnapshot.cs

```diff
@@ -654,6 +696,9 @@ protected override void BuildModel(ModelBuilder modelBuilder)
                     b.Property<string>("DirectionOfTravel")
                         .HasColumnType("nvarchar(max)");
 
+                    b.Property<int?>("DivisionFK")
```

Comment:
Use the name DivisionId for the foriegn key, keeping consistency with the existing tables